### PR TITLE
[14.0] [MIG] Project_status

### PR DESCRIPTION
**Pull request by Yoshiro009**
Standard Migration

### CODE REVIEW

**Review by emagdalenaC2i — APPROVED** [review]
LGTM

**Comment by Yoshiro009** [comment]
@emagdalenaC2i  thank.  for you read code  ^^

**Comment by skeller1** [comment]
@Yoshiro009: Maybe you could copy the test from #751 to get full test coverage. Thank you for contributing.

File: project_status/models/project.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from odoo import SUPERUSER_ID, api, fields, models
+
+
+class Project(models.Model):
+    _inherit = "project.project"
+
+    @api.model
+    def _read_group_status_ids(self, statuses, domain, order, access_rights_uid,_search):
+
```
Comment (skeller1) [inline project_status/models/project.py]: Please fix newlines with pre-commit run or manually.
group_expand needs method with `_my_method(self, values, domain, order)` ([REF 14 ORM docs](https://www.odoo.com/documentation/14.0/reference/orm.html#fields))
Runbot is failing at the moment with:
```
Error:
Odoo Server Error

Traceback (most recent call last):
  File "/.repo_requirements/odoo/odoo/addons/base/models/ir_http.py", line 237, in _dispatch
    result = request.dispatch()
  File "/.repo_requirements/odoo/odoo/http.py", line 682, in dispatch
    result = self._call_function(**self.params)
  File "/.repo_requirements/odoo/odoo/http.py", line 358, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/.repo_requirements/odoo/odoo/service/model.py", line 94, in wrapper
    return f(dbname, *args, **kwargs)
  File "/.repo_requirements/odoo/odoo/http.py", line 346, in checked_call
    result = self.endpoint(*a, **kw)
  File "/.repo_requirements/odoo/odoo/http.py", line 911, in __call__
    return self.method(*args, **kw)
  File "/.repo_requirements/odoo/odoo/http.py", line 530, in response_wrap
    response = f(*args, **kw)
  File "/.repo_requirements/odoo/addons/web/controllers/main.py", line 1359, in call_kw
    return self._call_kw(model, method, args, kwargs)
  File "/.repo_requirements/odoo/addons/web/controllers/main.py", line 1351, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/.repo_requirements/odoo/odoo/api.py", line 392, in call_kw
    result = _call_kw_model(method, model, args, kwargs)
  File "/.repo_requirements/odoo/odoo/api.py", line 365, in _call_kw_model
    result = method(recs, *args, **kwargs)
  File "/.repo_requirements/odoo/addons/web/models/models.py", line 91, in web_read_group
    expand_limit, expand_orderby)
  File "/.repo_requirements/odoo/addons/web/models/models.py", line 115, in _web_read_group
    orderby=orderby, lazy=lazy)
  File "/.repo_requirements/odoo/odoo/models.py", line 2243, in read_group
    result = self._read_group_raw(domain, fields, groupby, offset=offset, limit=limit, orderby=orderby, lazy=lazy)
  File "/.repo_requirements/odoo/odoo/models.py", line 2391, in _read_group_raw
    aggregated_fields, count_field, result, read_group_order=order,
  File "/.repo_requirements/odoo/odoo/models.py", line 1893, in _read_group_fill_results
    groups = getattr(self, field.group_expand)(groups, domain, order)
Exception

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/.repo_requirements/odoo/odoo/http.py", line 638, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/.repo_requirements/odoo/odoo/http.py", line 314, in _handle_exception
    raise exception.with_traceback(None) from new_cause
TypeError: _read_group_status_ids() missing 2 required positional arguments: 'access_rights_uid' and '_search'

```

Thank you for contributing.

**Review by skeller1 — CHANGES_REQUESTED** [review]

**Review by CasVissers-360ERP — APPROVED** [review]
Functional test.

**Review by skeller1 — APPROVED** [review]

**Review by newtratip — APPROVED** [review]
LGTM :+1:

**Comment by newtratip** [comment]
@OCA/project-service-maintainers I think, this should be merged.

**Pull request merged**